### PR TITLE
Install instructions update

### PR DESCRIPTION
**Pull request by ocefpaf**
Partially addresses #302

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/ioos/compliance-checker.svg)](https://travis-ci.org/ioos/compliance-checker)
 
-The IOOS Compliance Checker is a Python tool to check local/remote datasets against a variety of compliance standards. It is primarily a command-line tool (tested on OSX/Linux) and can also be used as a library import.
+The IOOS Compliance Checker is a Python tool to check local/remote datasets against a variety of compliance standards.
+It is primarily a command-line tool (tested on OS X/Linux) and can also be used as a library import.
```
Comment (ocefpaf) [inline README.md]: @lukecampbell I added a few line breaks for readability. If that scrambles the text too much let me know and I will remove this change.

File: README.md [hunk 2]
```diff
@@ -63,13 +72,13 @@ Running Compliance Checker on the dataset from: compliance_checker/tests/data/ru
 
 
 --------------------------------------------------------------------------------
-                    The dataset scored 100 out of 188 points                    
-                             during the acdd check
```
Comment (ocefpaf) [inline README.md]: There are a few blank spaces removal due to my editor setting. Let me know if that is a nuisance and I will edit the PR.

File: README.md [hunk 3]
```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
 
 ## Installation
 
-### Non-Python dependencies
+### Conda users
```
Comment (ocefpaf) [inline README.md]: Divided into two sections `conda` and `pip`. Both the `Ubuntu` instructions and how to install `conda` where added as links for further reading.

File: README.md [hunk 4]
```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
...
 ```
 
-The Python dependencies require several underlying system packages that most package managers should have. See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
+The Python dependencies require several underlying system packages that most package managers should have.
+See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
 
 Install dependencies, numpy must be installed on its own:
 
-```
-$ pip install numpy
```
Comment (ocefpaf) [inline README.md]: `pip install compliance-checker` should get `netcdf4` and that will get `numpy`.

File: README.md [hunk 5]
```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
...
 ```
 $ mkvirtualenv --no-site-packages compliance-checker
 $ workon compliance-checker
 ```
 
-The Python dependencies require several underlying system packages that most package managers should have. See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
+The Python dependencies require several underlying system packages that most package managers should have.
+See the [Installation](//github.com/ioos/compliance-checker/wiki/Installation) wiki page for more information.
 
 Install dependencies, numpy must be installed on its own:
```
Comment (daf) [inline README.md]: the numpy bit should be removed (since it appears you removed that install line)

File: README.md [hunk 6]
```diff
@@ -510,18 +479,18 @@ https://mmancusa.webex.com/mmancusa/ldr.php?RCID=e5e6fc5b6d218307f9eec863111e603
...
 - Ben Adams <ben.adams@rpsgroup.com>
 
-And many more testers!    
+And many more testers!
+
+Portions of the CF checker are based on Michael Decker's work, http://repositories.iek.fz-juelich.de/hg/CFchecker/
 
-Portions of the CF checker are based on Michael Decker's work, http://repositories.iek.fz-juelich.de/hg/CFchecker/    
- 
 <!--
```
Comment (daf) [inline README.md]: can this section be removed?

Comment (daf) [inline README.md]: er, this doesn't look right in the summary view, i'm talking about the commented out block of commands.

File: README.md [hunk 7]
```diff
@@ -362,91 +371,51 @@ Downloading cf-standard-names table version 35 from: http://cfconventions.org/Da
...
 
 Install dependencies, numpy must be installed on its own:
 
-```
-$ pip install numpy
+```shell
 $ pip install compliance-checker
 ```
 
-###<a name="anaconda"></a> Anaconda on linux
```
Comment (ocefpaf) [inline README.md]: "Folded" into the instructions above. Although maybe we should bring back the testing part. Not sure b/c the README is already too long.

**Pull request merged**